v3-sdk/routing/yarn.lock

```diff
@@ -2641,66 +2663,59 @@
+  resolved "https://registry.yarnpkg.com/@uniswap/token-lists/-/token-lists-1.0.0-beta.33.tgz#966ba96c9ccc8f0e9e09809890b438203f2b1911"
+  integrity sha512-JQkXcpRI3jFG8y3/CGC4TS8NkDgcxXaOQuYW8Qdvd6DcDiIyg2vVYCG9igFEzF0G6UvxgHkBKC7cWCgzZNYvQg==
 
-"@uniswap/universal-router-sdk@^1.1.0":
-  version "1.2.2"
-  resolved "https://registry.yarnpkg.com/@uniswap/universal-router-sdk/-/universal-router-sdk-1.2.2.tgz#26ece606279d1cd9823277b74cd43575caf160e9"
-  integrity sha512-L+MkvMiLyG1vfpEoLbe9xquSApSrfe1m0iLuIjgkZ4qR2Kb0NU8nZXNqeyTJmTZhNbYYIJA0AwRawAN+90NKIw==
+"@uniswap/universal-router-sdk@npm:@uniswapfoundation/universal-router-sdk@^2.1.0":
+  version "2.1.0"
+  resolved "https://registry.yarnpkg.com/@uniswapfoundation/universal-router-sdk/-/universal-router-sdk-2.1.0.tgz#5f89bbbb82d314f38d40ef3dac030d4a7f117112"
+  integrity sha512-iv1Ll2m0xDiyNdKmssRkVBuDN5oM8Tc4CbsnBIg9qgDVnJ2P2Y7TCUP25hCwdqtHXln1Tp+ZFy3OJp0DvnLsrA==
   dependencies:
     "@uniswap/permit2-sdk" "^1.2.0"
-    "@uniswap/router-sdk" "^1.4.0"
-    "@uniswap/sdk-core" "^3.1.0"
-    "@uniswap/universal-router" "1.1.1"
-    "@uniswap/v2-sdk" "^3.0.1"
-    "@uniswap/v3-sdk" "^3.9.0"
+    "@uniswap/router-sdk" "npm:@uniswapfoundation/router-sdk@^1.8.0"
+    "@uniswap/sdk-core" "npm:@uniswapfoundation/sdk-core@^5.0.0"
+    "@uniswap/universal-router" "1.5.1"
+    "@uniswap/v2-sdk" "npm:@uniswapfoundation/v2-sdk@^4.0.0"
+    "@uniswap/v3-sdk" "npm:@uniswapfoundation/v3-sdk@^4.0.1"
     bignumber.js "^9.0.2"
     ethers "^5.3.1"
 
-"@uniswap/universal-router@1.1.1", "@uniswap/universal-router@^1.0.1":
-  version "1.1.1"
-  resolved "https://registry.yarnpkg.com/@uniswap/universal-router/-/universal-router-1.1.1.tgz#9d197e20fe830aaccca468a1d7b178a6c4c30e26"
-  integrity sha512-bDdFuzKiuvatCH5RENEEQuS7IhD2TjAS6TEFtWGCF+uOTf23TJCQA0pA9PuPyN2qMPNsNNcQlHVdcyIUD+IkWg==
+"@uniswap/universal-router@1.5.1", "@uniswap/universal-router@^1.0.1":
+  version "1.5.1"
+  resolved "https://registry.yarnpkg.com/@uniswap/universal-router/-/universal-router-1.5.1.tgz#2ce832485eb85093b0cb94a53be20661e1aece70"
+  integrity sha512-+htTC/nHQXKfY/c+9C1XHMRs7Jz0bX9LQfYn9Hb7WZKZ/YHWhOsCZQylYhksieLYTRam5sQheow747hOZ+QpZQ==
+  dependencies:
+    "@openzeppelin/contracts" "4.7.0"
+    "@uniswap/v2-core" "1.0.1"
+    "@uniswap/v3-core" "1.0.0"
 
-"@uniswap/v2-core@1.0.1":
+"@uniswap/v2-core@1.0.1", "@uniswap/v2-core@^1.0.1":
   version "1.0.1"
   resolved "https://registry.yarnpkg.com/@uniswap/v2-core/-/v2-core-1.0.1.tgz#af8f508bf183204779938969e2e54043e147d425"
   integrity sha512-MtybtkUPSyysqLY2U210NBDeCHX+ltHt3oADGdjqoThZaFRDKwM6k1Nb3F0A3hk5hwuQvytFWhrWHOEq6nVJ8Q==
 
-"@uniswap/v2-sdk@^3.0.1":
-  version "3.0.1"
-  resolved "https://registry.yarnpkg.com/@uniswap/v2-sdk/-/v2-sdk-3.0.1.tgz#690c484104c1debd1db56a236e5497def53d698b"
-  integrity sha512-eSpm2gjo2CZh9FACH5fq42str/oSNyWcDxB27o5k44bEew4sxb+pld4gGIf/byJndLBvArR9PtH8c0n/goNOTw==
+"@uniswap/v2-sdk@npm:@uniswapfoundation/v2-sdk@^4.0.0":
```

Comment:
This is the line that is causing the GitHub bot to get confused.  It should be loading `@uniswapfoundation/v2-sdk` not `@uniswap/v2-sdk`.

Sorry about this mistake.

Reply:
Thanks @mikolalysenko. This should be the undeprecated package https://www.npmjs.com/package/@uniswapfoundation/v2-sdk

Reply:
I'll update this package to a newer version.